integrations/README.md

```diff
@@ -0,0 +1,10 @@
+This directory contains BigTest integration for Cypress. The reason why there is a separate monorepo is so that we can separate the dependencies of this directory from the BigTest core components in order to provide a better developer experience.
+
+## Development
+### Install/Build
+To develop packages inside the integrations directory, you can run `yarn install` and `yarn prepack` from the root of the BigTest repository and it will install/prepack both the integrations and the BigTest core components.
+
+Or you can `cd` into `integrations/` and run `yarn install` or `yarn prepack` to install and prepack just the packages inside `integrations/`.
```

Comment:
rather than have instructions, can we not supply an npm script to do this?

```
### Install/Build

yarn install-integrations
```

could we use `"yarn install:integrations"`

Reply:
Oh I already have `yarn install:integrations` so I guess I could say something like "if you only want to install/build the packages inside integrations, run `yarn install:integrations` and `yarn prepack:integrations`"

Reply:
> Oh I already have `yarn install:integrations` so I guess I could say something like "if you only want to install/build the packages inside integrations, run `yarn install:integrations` and `yarn prepack:integrations`"

I would have 1 command only.  Where possible, I would try and avoid a list of things to do.  

You could give them a script that combined those 2.

so...

```md
if you only want to install/build the packages inside integrations, run 

yarn install-and-build-integrations
```

then you have a script like

```sh
"scripts":  {
   "install-and-build-integrations": "yarn install:integrations && yarn prepack:integrations"
}
```

Reply:
Would it be misleading to just call it the build script? Install installs, prepack prepacks, build will install _and_ prepack?

Reply:
> Would it be misleading to just call it the build script? Install installs, prepack prepacks, build will install _and_ prepack?
> 
> ```
> yarn build:integrations
> ```

I like that.  Nice one!